backend/btrixcloud/colls.py

```diff
@@ -18,7 +18,9 @@
 class Collection(BaseMongoModel):
     """Org collection structure"""
 
-    name: str
+    id: UUID4
```

Comment:
The BaseMongoModel provides the id field already, so not needed here.
(Marked as optional there to allow auto-generating it).